### ISSUE TITLE
A Downshift depedency causes an import error when running in Vite

### ISSUE THREAD

**Issue by stdavis** [agrc/kitchen-sink#89]
Installing a `@ugrc/sherlock` into a freshly bootstrapped vite app causes the dev build to fail with the following message in the console...

```
Uncaught SyntaxError: The requested module '/node_modules/prop-types/index.js?v=e115d97d' does not provide an export named 'default' (at index.es.js?v=e115d97d:2:8)
```

[This older issue](https://github.com/downshift-js/downshift/issues/624) in the Downshift repo appears to be a similar problem but the suggested fix is a deprecated option in the associated [@rollup/plugin-commonjs](https://github.com/rollup/plugins/tree/master/packages/commonjs) package. I tried adding this plugin to the project and vite config but it didn't seem to have any effect.

**Comment by steveoh** [comment]
Does that mean that downshift should be a peer dependency?

**Comment by stdavis** [comment]
> Does that mean that downshift should be a peer dependency?

I wouldn't think so since it needs to be installed regardless. But maybe?

**Comment by steveoh** [comment]
this is because the jsx was not compiled and vite doesnt' optimize them. closed by #105
